layersvt/test/test_api_dump.cpp

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2023 The Khronos Group Inc.
+ * Copyright (c) 2023 Valve Corporation
+ * Copyright (c) 2023 LunarG, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Author: Christophe Riccio <christophe@lunarg.com>
+ */
+
+#include "layer_test_helper.h"
+
+#include <vulkan/vulkan_core.h>
+#include <vulkan/vulkan_beta.h>
+
+#include <gtest/gtest.h>
+
+#include <cstdarg>
+
+static const char* kLayerName = "VK_LAYER_LUNARG_api_dump";
+
+class ApiDumpTests : public VkTestFramework {
+   public:
+    ApiDumpTests(){};
+    ~ApiDumpTests(){};
+
+    static void SetUpTestSuite() {}
+    static void TearDownTestSuite(){};
+};
```

Comment:
I had no particularly desire to even keep the class but `VkTestFramework` is defined with `~VkTestFramework() = 0;` so we need a sub class and I don't want to break that behavior because that's how it behave on other repository. 

It's the same with SetUpTestSuite and TearDownTestSuite so the only think I could remove is the contructors (`ApiDumpTests()`) which I have removed.